src/python-frontend/python_converter.cpp

```diff
@@ -1575,18 +1611,29 @@ bool python_converter::convert()
 
   python_filename = ast_json["filename"].get<std::string>();
 
-  // Load memory models -----
-  std::stringstream model_path;
-  model_path << ast_json["ast_output_dir"].get<std::string>() << "/range.json";
-  std::ifstream model_file(model_path.str());
-  nlohmann::json model_json;
-  model_file >> model_json;
+  if (!config.options.get_bool_option("no-library"))
+  {
+    // Load operational models -----
```

Comment:
We could print a message to the ESBMC user saying that we are loading the operational model with the name of the model. What do you think?

Reply:
That sounds good. I'll make this modification.

Reply:
![image](https://github.com/esbmc/esbmc/assets/1334363/b316fa7c-4ed9-459c-899a-700bf9f3caa3)